nre/bpf/nodefw/nodefw-ebpf/.cargo/config.toml

```diff
@@ -0,0 +1,6 @@
+[build]
+target-dir = "../target"
+target = "bpfel-unknown-none"
+
+[unstable]
+build-std = ["core"]
```

Comment:
Do we need this file if we have the "xtask" job for building things?

Reply:
target-dir and target i think are accessed in the build scripts, but i'll check